x-pack/solutions/search/plugins/search_indices/public/utils/indices.ts

```diff
@@ -44,3 +48,28 @@ export function getFirstNewIndexName(startingIndexNames: string[], currentIndexN
   }
   return undefined;
 }
+
+export type HealthStatusStrings = 'red' | 'green' | 'yellow' | 'unavailable';
+export const healthColorsMap: Record<HealthStatusStrings, IconColor> = {
+  red: 'danger',
+  green: 'success',
+  yellow: 'warning',
+  unavailable: '',
+};
+
+export const healthColorsMapSelectable = {
+  red: 'danger',
+  RED: 'danger',
+  green: 'success',
+  GREEN: 'success',
+  yellow: 'warning',
+  YELLOW: 'warning',
+};
```

Comment:
As I can see we don't use it anywhere, propably you added this map before added `nomalizeHealth`